tsconfig.json

```diff
@@ -22,7 +22,7 @@
         "resolveJsonModule": true,
         "removeComments": true,
         "useUnknownInCatchVariables": false,
-        "types": ["@types/vscode-notebook-renderer/preload"]
+        "types": ["@types/vscode-notebook-renderer/preload", "webpack-env"]
```

Comment:
For better development experience (i.e. during editing we'll get code completions)